api/rpc/stack/parse.go

```diff
@@ -9,23 +9,13 @@ import (
 type serviceMap struct {
 	Image       string      `yaml:"image"`
 	Ports       []string    `yaml:"ports"`
-	Replicas    int         `yaml:"replicas"`
+	Replicas    int32       `yaml:"replicas"`
```

Comment:
I know it seems excessively large (I thought so as well), but the actual type used by docker for `replicas` is a `uint64`, and it's what I'm using for service right now.